### PR TITLE
Dgraph 1.1: modify tutorial

### PR DESCRIPTION
**Pull request by MichelDiz**
Need review.

### CODE REVIEW

**Comment by MichelDiz** [comment]
what happened to 1.1 branch? this need to be there, cuz remove comments and incompatible functions.v

File: content/moredata/2.md [hunk 1]
```diff
@@ -6,18 +6,14 @@ title = "Movies Schema"
...
 
-Dgraph queries and the visualization will help us understand the schema of the movies dataset.  
+Dgraph queries, types and the visualization will help us understand the schema of the movies dataset.  
 
 As you'll see in the following pages, we can view the data from a number of perspectives; considering for example directors, movies, genres or even locations as the primary starting point.
 
 Let's have a look from the perspective of directors.  Kathryn Bigelow directed the early 90's classic Point Break.  We'll begin with her and learn about the schema.
 
-Run the query.  `_predicate_` tells us that directors have a `name` and are connected to the movies they directed via `director.film`.
+Checking the last page you can se that directors are part of Person Type and have a `name` and are connected to the movies they directed via `director.film`.
```
Comment (martinmr) [inline content/moredata/2.md:15]: you can see

Comment (martinmr) [reply content/moredata/2.md:15]: are Part of the Person type, have a `name`, and are ....

File: content/moredata/2.txt [hunk 2]
```diff
@@ -1,6 +1,14 @@
-# TODO: Reformulate this topic according to the new Type System.
+
```
Comment (martinmr) [inline content/moredata/2.txt:1]: minor: remove blank line

File: content/schema/8.md [hunk 3]
```diff
@@ -6,6 +6,10 @@ title = "Predicate Query"
 weight = 8
 +++
 
+{{% notice "note" %}}
+  `_predicate_` doesn't work on Dgraph 1.1 but we plan to add a new function to do the same job.
```
Comment (martinmr) [inline content/schema/8.md:10]: maybe remove this. It's still not sure whether we will add an equivalent feature. Maybe say for now users can query the types (just like the schema) to get the fields that belong to that type.

File: content/moredata/2.md [hunk 4]
```diff
@@ -6,18 +6,14 @@ title = "Movies Schema"
 weight = 2
 +++
 
-Dgraph queries and the visualization will help us understand the schema of the movies dataset.  
+Dgraph queries, types and the visualization will help us understand the schema of the movies dataset.
```
Comment (martinmr) [inline content/moredata/2.md:9]: "types, and the visualization"

**Comment by paulftw** [comment]
@MichelDiz I've changed the tour build scripts to support the version switch.
For v1.1 specific patches please make changes to the branch named `dgraph-1.1.0`.

Sorry if I closed this PR/branch too soon. I thought I've merged it to 1.1

**Pull request closed without merging**